Makefile

```diff
@@ -0,0 +1,7 @@
+-include .postgres.env
+
+postgres_env:
+	echo "POSTGRES_USER=stringer\nPOSTGRES_PASSWORD=`pwgen -s 20`\nPOSTGRES_HOSTNAME=stringer-postgres\nPOSTGRES_DB=stringer\n" >> .postgres.env
+
+stringer_env:
+	echo "SECRET_KEY_BASE=`openssl rand -hex 64`\nENCRYPTION_PRIMARY_KEY=`openssl rand -hex 64`\nENCRYPTION_DETERMINISTIC_KEY=`openssl rand -hex 64`\nENCRYPTION_KEY_DERIVATION_SALT=`openssl rand -hex 64`\n\nDATABASE_URL=postgres://`echo ${POSTGRES_USER}`:`echo ${POSTGRES_PASSWORD}`@`echo ${POSTGRES_HOSTNAME}`/`echo ${POSTGRES_DB}`\nFETCH_FEEDS_CRON='*/5 * * * *'\nCLEANUP_CRON='0 0 * * *'\n" >> .stringer.env
```

Comment:
If we go the Ruby/rake way, we could also use erb to generate these files. E.g.:

```erb
# docker/.stringer.env.erb
SECRET_KEY_BASE=<%= `openssl rand -hex 64`.strip %>
ENCRYPTION_PRIMARY_KEY=<%= `openssl rand -hex 64`.strip %>
ENCRYPTION_DETERMINISTIC_KEY=<%= `openssl rand -hex 64`.strip %>
ENCRYPTION_KEY_DERIVATION_SALT=<%= `openssl rand -hex 64`.strip %>

DATABASE_URL=postgres://<%= "#{ENV['POSTGRES_USER']}:#{ENV['POSTGRES_PASSWORD']}@#{ENV['POSTGRES_HOSTNAME']}/#{ENV['POSTGRES_DB']}" %>
FETCH_FEEDS_CRON='*/5 * * * *'
CLEANUP_CRON='0 0 * * *'
```

I think the rake task would look something like:

```rb
namespace :env do
  task :generate_stringer do
    File.write("./.stringer.env", `erb ./docker/.stringer.env.erb`)
  end
end
```

That way it's a little more consistent with Rails conventions. Being a Ruby guy, I'm a bit biased, but I also find that to be a bit more readable.

Reply:
I agree, the `\n` delimited stuff isn't super readable. I could split it into multiple echo calls, but as you've mentioned, `rake` seems to be the standard so let's go with that :) though given my inexperience with Ruby there might be a few more comments from your part to make everything look right, heh.

Reply:
Thanks for this code snippet! This made it a lot easier to make the changes.